variables.tf

```diff
@@ -168,3 +168,9 @@ variable "auto_minor_version_upgrade" {
   type        = bool
   default     = false
 }
+
+variable "bastion_ingress_cidr_prefix" {
+  description = "Prefix of local_ip used as ingress for bastion"
+  default     = 32
+  type        = number
+}
```

Comment:
@vamshidhar-rapolu This is a bit picky, but please add newline to end of file

Reply:
Sure